Mathlib/Computability/RegularExpressions.lean

```diff
@@ -319,10 +319,14 @@ theorem star_rmatch_iff (P : RegularExpression α) :
         constructor
         · simp [hs, hsum]
         · intro t' ht'
-          cases' ht' with ht' ht' -- porting note: last ht' doesn't take, need to `rename_i` below
-          · simp only [ne_eq, not_false_iff, true_and, rmatch]
+          -- porting note: was `cases' with ht' ht'` but a `cases'` bug
+          -- caused the last `ht'` not to take, requiring `rename_i`
+          -- moved to explicit cases.
```

Comment:
I think you can just remove the porting note, at this point. `cases'` is only intended as a stopgap measure, so once you've rewritten it using structured cases this file doesn't need revisiting.